steam_auto_cracker_gui.py

```diff
@@ -73,27 +72,47 @@ def DoRequest(self):
             else:
                 self.req = req
 
-    def select_folder():
-        global folder_path
-        folder_path = filedialog.askdirectory()
-        if not os.path.isdir(folder_path):
-            update_logs("\nNo folder has been selected")
+    def handle_folder_selection(folder_path=None, event=None):
+        global last_dropped_folder
+        last_dropped_folder = config["Preferences"].get("last_dropped_folder", "")
+        # Reset and hide UI elements related to folder selection and game cracking
+        def reset_folder_selection_ui():

```

Comment:
Why is `reset_folder_selection_ui` declared inside `handle_folder_selection`? Seems a bit weird

Reply:
Thanks, and it might've been an oversight. But I believe it's because `reset_folder_selection_ui` is only relevant within `handle_folder_selection` and doesn't need to be accessed elsewhere. I believe my reasoning is I thought nesting it helped keep the related code together, thus improving readability and structure. It may be better to change it, especially if it one day is used elsewhere. Please feel free to change it, or I can!

Reply:
Got it, I don't know python enough to judge this, but it seems to work, so all good. If it should be changed, it'll be doable later, as long as this works, I'm fine with it